charts/komodor-agent/templates/admission-controller/rbac.yaml

```diff
@@ -73,6 +74,15 @@ rules:
       - pods/resize
     verbs:
       - patch
+  - apiGroups:
+      - keda.sh
+    resources:
+      - scaledobjects
```

Comment:
Worth verifying if you need an API version. If you do - you may need to specify multiple versions or even `"*"`.

Reply:
As you already told me it's unnecessary and actually impossible, I'm approving